models.py

```diff
@@ -0,0 +1,35 @@
+import os
+import random
+orders={}
+class order:
+
+    def __init__(self):
+        self.orders=orders
+    def make_order(self, new_order):
+        x = random.randint(1, 20)
+        self.orders[x]= new_order
+        return new_order
+
+    def list_orders(self):
+
+        return {"order_details": self.orders, "status_code":200}
+
+    def specific_order(self,orderId):

```

Comment:
Since you are storing your orders in a `dict` you could easily retrieve one by  using `get` , something like this
```python
 def specific_order(self, orderId):
        order = self.orders.get(orderId)
        if order:
            return order
        return ("order not found")
 ```